Makefile

```diff
@@ -810,21 +788,15 @@ partialclean::
 
 .PHONY: runtop
 runtop:
-ifeq "$(UNIX_OR_WIN32)" "unix"
-	$(MAKE) runtime
-	$(MAKE) coreall
-	$(MAKE) ocaml
-else
 	$(MAKE) core
 	$(MAKE) ocaml
-endif
```

Comment:
Again, if we're in the business of tidying these, why not do the minimum necessary to start the toplevel:

```make
	$(MAKE) coldstart
	$(MAKE) ocamlc
	$(MAKE) ocaml
```

Reply:
Note that the toplevel is started with option `-I otherlibs/`[`win32`]`unix` so we should probably also build `otherlibs` for consistency.